pyproject.toml

```diff
@@ -62,7 +62,7 @@ exclude_lines = [
 [tool.pyright]
 include = ["examples", "hikari"]
 pythonVersion = "3.8"
-typeCheckingMode = "strict"
+typeCheckingMode = "basic"
```

Comment:
This should never be basic, this should be strict if we can keep it that way. It might error for now but we want to work towards being pyright compliant too

Reply:
Yeah my bad, I made these changes because of how damn unbearable it was to deal with strict pyright in a codebase made for mypy.